torch/csrc/utils/python_arg_parser.cpp

```diff
@@ -1260,7 +1260,7 @@ void FunctionParameter::set_default_str(const std::string& str) {
   } else if (type_ == ParameterType::SCALARTYPE) {
     if (str == "None") {
       default_scalartype = at::ScalarType::Undefined;
-    } else if (str == "torch.int64") {
+    } else if (str == "torch.int64" || str == "at::kLong") {
```

Comment:
It's a little tricky.

But, It can handle below cases.
```
// torch/csrc/autograd/generated/python_variable_methods.cpp
// torch/csrc/autograd/generated/python_torch_functionsEverything.cpp
// torch/csrc/autograd/generated/python_torch_functions_2.cpp
static PythonArgParser parser({
   "sort(*, bool? stable, Dimname dim, bool descending=False)",
   "sort(*, bool? stable, int64_t dim=-1, bool descending=False, ScalarType indices_dtype=at::kLong, bool dynamic_indices_dtype=False)",
   "sort(Dimname dim, bool descending=False)",
   "sort(int64_t dim=-1, bool descending=False, ScalarType indices_dtype=at::kLong, bool dynamic_indices_dtype=False)",
}, /*traceable=*/true);
```